env.js

```diff
@@ -0,0 +1,15 @@
+// central place to read and expose env vars and to assign default values
+// TODO: consider move other references of process.env to here
+
+// read env vars
+const result = require('dotenv').config();
+
+if (result.error) {
+  throw result.error
+}
+const env = result.parsed;
+
+module.exports = {
+  region: env.REGION || 'bonn',
+  cypressBrowserWindow: env.CYPRESS_BROWSER_WINDOW
+}
```

Comment:
Is it possible to invoke dotenv directly in the Cypress code so that this module can be removed and all env vars are accessible from Cypress code directly?

Reply:
I am not a fan of this pattern I guess.. It moves env vars from `process.env`, where they are expected, to a module. Then, this module a) doesn't include all env vars available in the main application code and b) changes the names of env vars. Both will be sources of errors in the future.

Reply:
Just a proposal, it can easily be removed.
Just let me underline the PROs of this aproach once more:

1. You have one central file which looks up keys in `process.env`, the knowledge is not cluttered. 
1. We gain Intellisense by explicitly stating those properties in a module instead of assuming global properties.
1. We can assign default values and add documentation to those properties in a central place.

Regarding your points
a) Remaining env vars can easily be added.
b) Names can easily be changed using IDE refactoring tools, easier than when we only rely on an opaque `process.env`.

Please consider this pattern once more, but in the end I am also fine to remove the `env.js`.

Reply:
For the reasons that you mention I think we should try and use env vars as little as possible.  Env vars should serve three purposes:

a) select the actual application configuration to be loaded (which is what the `REGION` and `BACKEND` env vars do)
b) allow each developer to adapt the application to their development environment (which is what e.g. `CYPRESS_BROWSER_WINDOW` does)
c) hold application secrets that should not be checked into code

Everything else should go into a config file anyway, so we won't ever have lots and lots of env vars. I can understand the advantages that you listed but I think the downsides weigh heavier in this case. 

By the way, you can probably also configure environment variables in your IDE. In my IDE for example, I can create any number of launch scripts that run the same package.json script but with different env vars. Maybe I don't fully understand the advantage that you get in IntelliSense from defining these variables in a module. Is it possible to gain your pro-point 2) without this module?

Reply:
Hey @ciex , I might be on thin ice here but I'll give it another shot.

I propose to use a little JavaScript that _wraps_ (read + assignment of default values) defined environment variables (no matter how, wether this happens in the IDE, a local console or by a CI script). It is not ment to _replace_ the usage of environment vars. An `env.js` is only a file to
* explicitly declare what env vars we expect
* declare fallback values
* facilitate consumption in our code base (intellisense)

Anyways, I like this pattern because I got to know its advantages while working with https://github.com/w3tecch/express-typescript-boilerplate/blob/develop/src/env.ts,
but I don't want to enforce my taste here. 
I am fine to remove the wrapper.